apps/reference/docs/guides/auth/auth-mfa.mdx

```diff
@@ -0,0 +1,570 @@
+---
+id: auth-mfa
+title: 'Multi-Factor Authentication'
+description: Add an additional layer of security to your apps with Supabase Auth multi-factor authentication.
+---
+
+import { Mermaid } from 'mdx-mermaid/Mermaid';
+
+:::note
+Multi-Factor Authentication is in early access preview only.
+:::
+
+Multi-factor authentication (MFA) or sometimes also called two-factor
+authentication (2FA) is a way to add an additional layer of security to your
+application by requiring users to verify their identity through one or more
+additional verification steps.
+
+It is considered a best practice for all but the simplest of applications.
+
+Users who use weak passwords or have compromised social login accounts are
+prone to malicious account takeovers. These can be prevented by using
+multi-factor authentication because they require the user to provide proof of
+both of these:
+
+- Something they know.  
+Password, or access to a social-login account.
+- Something they have.  
+Access to an authenticator app (a.k.a. TOTP), mobile phone or recovery code.
+
+## Overview
+
+Supabase Auth implements only TOTP multi-factor authentication. This type of
```

Comment:
/nit should we define TOTP? Think the adjective might be more well known then the full term though
```suggestion
Supabase Auth implements only Time-based One Time Factor(TOTP) multi-factor authentication. This type of
```